uniflow-core/src/main/kotlin/io/uniflow/core/logger/EmptyLogger.kt

```diff
@@ -0,0 +1,19 @@
+package io.uniflow.core.logger
+
+import io.uniflow.core.flow.data.UIEvent
+import io.uniflow.core.flow.data.UIState
+
+/**
+ * Logger to ... log nothing
+ */
+class EmptyLogger : Logger {
```

Comment:
Some optional ideas:

1. This class could be `open` to make it extensible by users who want to log only if a subset of these functions is called. Downside: their extension would be an `EmptyLogger` that is not empty. But in the end it's also a `Logger`.
1. Alternatively, make the `Logger` interface functions empty by default. Downside of that is that a user might forget to override the empty defaults.

Reply:
Not sure it's really needed here, if you want a new logger it's faily easy to write.